File: 75-sort-colors/75-sort-colors.cpp
class Solution {
public:
    void sortColors(vector<int>& nums) {
        int ones = 0, zeros = 0, twos = 0;
        for(int i:nums){
            if(i==0) zeros++;
            else if(i==1) ones++;
            else twos++;
        }
        for(int i=0; i<zeros; i++){
            nums[i] = 0;
        }
        for(int i=zeros; i<ones+zeros; i++){
            nums[i] = 1;
        }
        for(int i=ones+zeros; i<nums.size(); i++){
            nums[i] = 2;
        }
    }
};

File: 75-sort-colors/75-sort-colors.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        zero = 0
        one = 0
        two = 0
        for i in nums:
            if i==0:
                zero+=1
            elif i==1:
                one+=1
            else:
                two+=1
        for i in range(0,zero):
            nums[i] = 0
        for i in range(zero,len(nums)-two):
            nums[i] = 1
        for i in range(len(nums)-two,len(nums)):
            nums[i] = 2
        